src/microbe_stage/Microbe.cs

```diff
@@ -395,6 +395,9 @@ public void ApplySpecies(Species species)
             MulticellularCellType = earlyMulticellularSpecies.Cells[0].CellType;
 
             cachedMulticellularSpecies = earlyMulticellularSpecies;
+
+            if (IsPlayerMicrobe)
+                GD.Print("Applying multicellular species type to player");
```

Comment:
This print is not necessary. If you want to confirm that the player also gets applied the new species add a check in MoveToMulticellular that prints an error if the player wasn't processed.